pyiode/pyiode_util.pyx

```diff
@@ -34,6 +38,24 @@ def pystr(cstr) -> str:
     if cstr is None: return None
     return cstr.decode("cp850")
 
+# Convert None, a list or str or an str to a str
+def arg_to_str(arg, sep: str = ' ') -> str:
+    '''Convert a string, a list of strings or None to a string.'''
+    if isinstance(arg, list):
```

Comment:
if isinstance(arg, Iterable):

Reply:
Does not work. Same reason as in pyiode_estim.